public/assets/main.css

```diff
@@ -425,3 +425,24 @@ div#game:focus {
     float: left;
   }
 }
+
+table.create-game-table thead tr th {
```

Comment:
why did you need to add this?

Reply:
Without `text-align: left`:

![Screenshot 2022-11-27 122410](https://user-images.githubusercontent.com/1045173/204155680-7d5c7118-14c0-43b7-8f06-bbbfaef7a9ec.png)

With it:

![Screenshot 2022-11-27 122312](https://user-images.githubusercontent.com/1045173/204155686-7c212003-5b20-45a5-a529-be982e351b27.png)

Reply:
why didn't you just add it inline? why does it need to be in the css?

Reply:
Original PR was too long ago for me to remember the why, but it works just fine inline 🙃 Fixed in 729f04a55